### PR TITLE
cleaning up markdown and adding a getting started section

### LINKED ISSUE #12: Readme does not contain getting started
It is nice to have a section of the readme that describes how to get up and running with the codebase. Seeing as this project uses submodules, it might be problematic for new contributors to figure out how to pull ALL the source.

### PR DESCRIPTION
**Pull request by NotMyself**
Fixes #12 

I cleaned up the readme making it valid markdown and added a getting started section to help new contributors get up and running with the source.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,26 +4,43 @@ CV for [Curriculum Vitae](https://en.wikipedia.org/wiki/Curriculum_vitae) is a w
...
+## Getting Started
+
+1. Install [.NET Core SDK 1.0](https://www.microsoft.com/net/core).
+1. Install [Visual Studio Code](https://code.visualstudio.com/), the [Insiders Edition](https://code.visualstudio.com/insiders) is highly recommended.
+1. Install Docker for [Windows](https://download.docker.com/win/beta/InstallDocker.msi), [Mac](https://download.docker.com/mac/beta/Docker.dmg) or [Linux](https://docs.docker.com/engine/installation/linux/).
+1. Clone the repository: `git clone https://github.com/SouthSoundDevelopers/CVPZ.git`.
+1. Change directory into the cloned repository `cd CVPZ`.
+1. Restore Submodules `git submodule update --init --recursive`.
+1. Restore packages: `dotnet restore`.
+1. Open the project in VSCode `code-insiders .` or `code .` depending on what version you chose to install.
```
Comment (emragins) [inline README.md:46]: I followed this and it worked great so far, but now I'm stuck lacking knowledge of .net core -- how do I build/run it and make sure that I have a "good" version before playing with it more?

Comment (emragins) [reply README.md:46]: PS. Yeah, I could probably figure it out, but that ain't the point.

Comment (NotMyself) [reply README.md:46]: Agreed. I think the layout of the project needs to firm up more before we can add info on running the application. Also I think @CarBar is working on docker files, which is why I added the step to install docker.

Comment (CarBar) [reply README.md:46]: I am working on figuring out docker. I will add the docker startup when I get it working. Thanks for adding the install.

**Pull request merged**